bismuthcore/bismuthcore/legacy/peershandler.py

```diff
@@ -470,9 +472,11 @@ def can_connect_to(self, host, port):
         if host_port in self.connection_pool:
             return False  # Already connected to
         try:
-            tries, timeout = self.tried[host_port]
-        except:
-            tries, timeout = 0, 0  # unknown host for now, never tried.
+            tries, timeout = self.tried[host_port]  # noqa: F841
+        except Exception as e:
+            # unknown host for now, never tried.
+            self.app_log.debug(f"Error getting tries for {host_port}: {e}")
```

Comment:
This is not an error, this is the normal working for an host we don't know yet.
This is ok and should not generate a log.

The pythonic way would be a use a defaultDict with (0,0) or .get(host_port, (0,0)) and get rid of the try /except alltogether.

Reply:
See comments regarding `legacy/`